man/org.freedesktop.systemd1.xml

```diff
@@ -1898,6 +1898,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       readonly s CollectMode = '...';
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
       readonly as Refs = ['...', ...];
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
```

Comment:
"const" is wrong. A unit can be loaded and then be started/stopped multiple times and for each cycle the activation info should be able to change